edc-extensions/ssi/jws2020-crypto-suite/README.md

```diff
@@ -0,0 +1,67 @@
+# JsonWebSignature2020
+
+This module extends the [iron-verifiable-credentials library](https://github.com/filip26/iron-verifiable-credentials),
+which we use in conjunction with [titanium-ld](https://github.com/filip26/titanium-json-ld/) with an implementation for
+the [JsonWebSignature2020](https://www.w3.org/community/reports/credentials/CG-FINAL-lds-jws2020-20220721) crypto suite.
+
+## Technical aspects
+
+This implementation is actually mostly glue code between the `iron-verifiable-credentials` lib and the
+well-known [Nimbus JOSE lib](https://connect2id.com/products/nimbus-jose-jwt), as all cryptographic primitives are taken
+from Nimbus.
+
+VerifiableCredentials and VerifiablePresentations are processed as JSON(-LD) objects, so some familiarity with JSON-LD
+is required.
+The entrypoint into the cryptographic suite is the `Vc` class, which allows signing/issuing and verifying JSON-LD
+structures. The following samples use explicit types for clarity. These are just some illustrative examples, please
+check the `IssuerTests` and the `VerifierTests` for more comprehensive explanations.
+
+### Sign a VC
+
+```java
```

Comment:
could be this code snipped formatted correctly?

Reply:
jeez, IDK whatever ungodly code formatter mangled this... 😱